.editorconfig

```diff
@@ -12,6 +12,9 @@ indent_size = 2
 [*.php]
 indent_size = 4
 
+[*.scss]
+indent_size = 4
```

Comment:
Can we agree on 2 spaces ? 😬
This is what we are using right now.

Reply:
Ok, I think our `scss` files are half 2 / half 4

I will clean this.